modules/gapi/include/opencv2/gapi/infer.hpp

```diff
@@ -531,7 +531,11 @@ typename Net::Result infer(Args&&... args) {
 }
 
 /**
- * @brief Special network type
+ * @brief Special network type.
```

Comment:
How it is special? Why it is special?

I'd name call it `Generic network type: input and output layers are configured dynamically at runtime`.

Reply:
Done